wolfcrypt/src/rsa.c

```diff
@@ -580,7 +580,8 @@ static int wc_RsaPad_ex(const byte* input, word32 inputLen, byte* pkcsBlock,
 /* UnPad plaintext, set start to *output, return length of plaintext,
  * < 0 on error */
 static int wc_RsaUnPad_OAEP(byte *pkcsBlock, unsigned int pkcsBlockLen,
-             byte **output, int hType, int mgf, byte* optLabel, word32 labelLen)
+                            byte **output, enum wc_HashType hType, int mgf,
+                            byte* optLabel, word32 labelLen)
 {
     int hLen;
     int ret;
```

Comment:
My testing all passed for this change, but did find one existing error when building with G++ on line 597 with the XMALLOC. You just need to cast the XMALLOC result to (byte*). Once fixed I'll merge it all in.

wolfcrypt/src/rsa.c:597:9: error: assigning to 'byte *' (aka 'unsigned char *') from incompatible type 'void *'
    tmp = XMALLOC(pkcsBlockLen, NULL, DYNAMIC_TYPE_TMP_BUFFER);
        ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~